www-apache/mod_security/mod_security-2.9.6.ebuild

```diff
@@ -0,0 +1,125 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+LUA_COMPAT=( lua5-{1..3} )
+
+inherit autotools apache-module lua-single
+
+MY_PN=modsecurity
+MY_P=${MY_PN}-${PV}
+
+DESCRIPTION="Application firewall and intrusion detection for Apache"
+HOMEPAGE="https://github.com/SpiderLabs/ModSecurity"
+SRC_URI="https://github.com/SpiderLabs/ModSecurity/releases/download/v${PV}/${MY_P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+apache2 doc fuzzyhash geoip jit json lua mlogc"
+
+REQUIRED_USE="apache2
```

Comment:
Just make it always-on then?

Reply:
Hello @thesamesam, how can that be expressed in the ebuild?

Reply:
Drop IUSE=apache2 & REQUIRED_USE, pass --enable-apache2 etc instead.

Alternatively, if you want to make it optional in future but it just doesn't work right now, we can add it in profiles/base/package.use.force. What do you prefer/intend?

Reply:
That's not passed to mod_security itself but used in https://github.com/gentoo/gentoo/blob/master/eclass/depend.apache.eclass#L193. 

I don't want to call the internals:
```
pkg_setup() {
  init_apache2
  init_apache2_late
```

But these are called from depend.apache only when apache2 is in IUSE (https://github.com/gentoo/gentoo/blob/master/eclass/depend.apache.eclass#L171).

There are 2 options:
1) https://gitweb.gentoo.org/repo/gentoo.git/commit/?id=ad49fd833d609fe1a93dd7b47df70a88dddd03d4
2) https://bugs.gentoo.org/616612#c13

Reply:
Ah, now I understand! Sorry, it seems obvious now you explained it.

It's fine if you use my method or if you call internals for now, as the eclass is broken anyway. I guess my preference is my method.

Reply:
This is not a good solution, we should avoid using REQUIRED_USE as much as possible. As I see it, it's really no-op here, just an annoyance.

Reply:
Just a side note, I started by the previous ebuild by using the internal calls (init_apache2 && init_apache2_late), but the CI report failed.

Reply:
Just checking https://gitweb.gentoo.org/repo/gentoo.git/tree/www-apache/mod_fastcgi_handler/mod_fastcgi_handler-0.6-r3.ebuild and it's also using the internal calls.

https://gitweb.gentoo.org/repo/gentoo.git/commit/www-apache/mod_fastcgi_handler?id=ad49fd833d609fe1a93dd7b47df70a88dddd03d4
https://gitweb.gentoo.org/repo/gentoo.git/commit/www-apache/mod_fastcgi_handler?id=9f513ee89c046aef6ca3c6f43ec999da0d2b4946